src/plugins/dev_tools/public/application.tsx

```diff
@@ -175,16 +175,16 @@ function DevToolsWrapper({
           <div className="devAppDataSourcePicker">
             <EuiComboBox
               aria-label={i18n.translate('devTools.devToolWrapper.DataSourceComboBoxAriaLabel', {
-                defaultMessage: 'Select a Data Source',
+                defaultMessage: 'Select a data source',
               })}
               placeholder={i18n.translate('devTools.devToolWrapper.DataSourceComboBoxPlaceholder', {
-                defaultMessage: 'Select a Data Source',
+                defaultMessage: 'Select a data source',
               })}
               singleSelection={{ asPlainText: true }}
               options={dataSources}
               selectedOptions={selectedOptions}
               onChange={onChange}
-              prepend="DataSource"
+              prepend="Data source"
```

Comment:
A little concern that why did we make the prepend content a constant text instead of wrapping it with `i18n`? Is `Data source` a dedicated terms that should not be translated?

Reply:
I think this `prepend` is something else. I couldn't find docs on OUI that made sense to me. @BSFishy do you know what `prepend` does?

Reply:
Id i18n this text too